### DISCUSSION TITLE
Support for sensorbox 1 or 1.5

### DISCUSSION THREAD

**Discussion by Ferforfaen** [serkri/SmartEVSE-3#111] [General]
I just changed my Smartevse 2.1 to a 3.0, and to my surprise. my old sensorbox willnot communicate with the 3.0 hardware. I am not sure if i have the 1.0 og 1.5 sensor version. 

Will it be possible to make this setup work?

**Comment by dingo35** [comment]
AFAIK not a lot of experience with SmartEVSEv2 and/or old sensorbox here; better check this on the original smartevse3 github from Stegen: 
https://github.com/SmartEVSE/SmartEVSE-3

**Comment by Ferforfaen** [comment]
I contacted Stegen, and he supplied me an update .hex file for the PIC controller. Borrowed a programmer and got the problem fixed.
